O3-1.tex

```diff
@@ -22,37 +22,40 @@
 
 \title{The LSST Data Management System}
 \author{
-Mario~Juric,$^1$
-Jeffrey~Kantor,$^2$
```

Comment:
Better than copying and pasting my changes ;).
I can redo the branch and force push, would you like me to?